app/views/menus/_meal.html.erb

```diff
@@ -0,0 +1,5 @@
+<ul>
+  <li><%= meal.name %></li>
+  <li><%= meal.course %></li>
+  <li>Costs: $<%= meal.price %></li>
```

Comment:
Use [number_to_currency](http://api.rubyonrails.org/classes/ActionView/Helpers/NumberHelper.html#method-i-number_to_currency) for this.